cura/Arranging/Nest2DArrange.py

```diff
@@ -114,10 +123,12 @@ def findNodePlacement(nodes_to_arrange: List["SceneNode"], build_volume: "BuildV
 def createGroupOperationForArrange(nodes_to_arrange: List["SceneNode"],
                                    build_volume: "BuildVolume",
                                    fixed_nodes: Optional[List["SceneNode"]] = None,
-                                   factor = 10000,
-                                   add_new_nodes_in_scene: bool = False)  -> Tuple[GroupedOperation, int]:
+                                   factor: int = 10000,
+                                   add_new_nodes_in_scene: bool = False,
```

Comment:
Another booltrap and one that we already had. Whoops...

Reply:
fixed in https://github.com/Ultimaker/Cura/pull/15496/commits/118f49a0520bdd1a3983bbf33b49c3ed8a645fb1